src/input_cp2k_dft.F

```diff
@@ -6331,6 +6331,14 @@ SUBROUTINE create_print_simple_grid(section)
       CALL section_add_keyword(print_key, keyword)
       CALL keyword_release(keyword)
 
+      CALL keyword_create(keyword, name="FOLD_COORD", &
+                          description="Activates printing folded coordinates corresponding "// &
+                          "to the simple grid. Used as imput for external programs.", &
```

Comment:
Typo: imput -> input

Reply:
Corrected.

Reply:
Looks like you have not yet pushed the corrected commit. Try:
```
git add  src/input_cp2k_dft.F
git commit --amend 
git push --force origin embedding
```

Reply:
Thanks! I was not aware of the --amend option.